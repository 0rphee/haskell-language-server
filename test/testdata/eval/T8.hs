module T8 where

-- >>> noFunctionWithThisName
