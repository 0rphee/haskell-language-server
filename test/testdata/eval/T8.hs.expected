module T8 where

-- >>> noFunctionWithThisName
-- Variable not in scope: noFunctionWithThisName
